### ISSUE TITLE
Error when vue model changes

### ISSUE THREAD

**Issue by juandfranco** [vueform/multiselect#191]
Hello, greetings, I make API calls that return data through hooks, these are assigned to my model and when they change it had a default value, but the model changes and does not have this value inside, the application breaks, runtime- core.esm-bundler.js?d2dd:656 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'emitsOptions') , is there a way that when v-model values change dynamically the first option is selected by default or clear before loading data, now I have to call everywhere I use the component

MyMultiselect.value.clear();

**Comment by afduarte** [comment]
I believe I have a similar issue. when the :value I'm passing to the component changes I get this in the console:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'label')
    at eval (multiselect.js?a306:formatted:1602)
    at renderSlot (runtime-core.esm-bundler.js?5c40:5818)
    at eval (multiselect.js?a306:formatted:1602)
    at renderList (runtime-core.esm-bundler.js?5c40:5735)
    at Proxy.L.render (multiselect.js?a306:formatted:1602)
    at renderComponentRoot (runtime-core.esm-bundler.js?5c40:434)
    at ReactiveEffect.componentUpdateFn [as fn] (runtime-core.esm-bundler.js?5c40:4267)
    at ReactiveEffect.run (reactivity.esm-bundler.js?a1e9:160)
    at callWithErrorHandling (runtime-core.esm-bundler.js?5c40:6666)
    at flushJobs (runtime-core.esm-bundler.js?5c40:6905)
```

I am using the tags mode and createTags=true. Is there any way a tag can be created if the value passed doesn't exist in the options?

**Comment by amaelftah** [comment]
@juandfranco @afduarte after a long time of searching I found the solution for the problem is using key and force rendering the component 

for example 
<Multiselect :key="post.id"/> and the post id is changed whenever the value you are passing is changing

**Comment by adamberecz** [comment]
I believe this has been resolved in `2.3.4` - if not please reopen with an example.
